README.md

```diff
@@ -13,11 +13,17 @@ It contains information about his projects, skills, experience and includes his
 - JAVASCRIPT
 
 
```

Comment:
this is optional you should not use JAVASCRIPT anymore!  please instead use HTML and CSS. thankyou. 
Happy codding